documentation/03-basic-crud.markdown

```diff
@@ -33,79 +36,40 @@ INSERT INTO author (first_name, last_name) VALUES ('Jane', 'Austen');
 
 ## Reading Object Properties ##
 
-Propel maps the columns of a table into properties of the generated objects. For each property, you can use a generated getter to access it.
+You can read values from the Perpl object in the same way that you set them.  Instead of `setXXX()` we use `getXXX()`
 
 ```php
 <?php
-echo $author->getId();        // 1
 echo $author->getFirstName(); // 'Jane'
 echo $author->getLastName();  // 'Austen'
 ```
 
-The `id` column was set automatically by the database, since the `schema.xml` defines it as an `autoIncrement` column. The value is very easy to retrieve once the object is saved: just call the getter on the column phpName.
-
-These calls don't issue a database query, since the `Author` object is already loaded in memory.
-
-You can also export all the properties of an object by calling one of the following methods: `toArray()`, `toXML()`, `toYAML()`, `toJSON()`, `toCSV()`, and `__toString()`:
-
-```php
-<?php
-echo $author->toJSON();
-// {"Id":1,"FirstName":"Jane","LastName":"Austen"}
-```
-
->**Tip**For each export method, Propel also provides an import method counterpart. So you can easily populate an object from an array using `fromArray()`, and from a string using any of `fromXML()`, `fromYAML()`, `fromJSON()`, and `fromCSV()`.
-
-There are a lot more useful methods offered by the generated objects. You can find an extensive  list of these methods in the [Active Record reference](/documentation/reference/active-record.html).
-
 ## Retrieving Rows ##
 
 Retrieving objects from the database, also referred to as _hydrating_ objects, is essentially the process of executing a SELECT query against the database and populating a new instance of the appropriate object with the contents of each returned row.
 
-In Propel, you use the generated Query objects to retrieve existing rows from the database.
+In Perpl, you use the generated Query objects to retrieve existing rows from the database.
 
 ### Retrieving by Primary Key ###
 
-The simplest way to retrieve a row from the database, is to use the generated `findPK()` method. It simply expects the value of the primary key of the row to be retrieved.
+The simplest way to retrieve a row from the database, is to use the generated `findPk()` method. This assumes you know the primary key of the record you want to retrieve and also that a primary key identified a unique record.  In the following example we are going to presume that our Author table has three fields, the first being the ID field that contains a unique numeric key.
```

Comment:
Should it be "assumes you know the *value of the* primary key of the record..."?

Primary key, by definition identifiers a unique record, I think that part should be removed.